servers/curated-corpus-api/src/shared/resolvers/fields/SectionStatus.ts

```diff
@@ -36,7 +36,18 @@ export function computeSectionStatus(section: Section): SectionStatus {
 
   // 2. If section has a startDate (custom section logic)
   if (section.startDate) {
-    const startDate = DateTime.fromJSDate(section.startDate).setZone(timeZone).startOf('day');
+    // Convert JS Date → Luxon DateTime in the section's local timezone,
+    // treating the JS Date as a plain calendar date (not a UTC timestamp).
+    // JS Date months are 0-based (January = 0), but Luxon expects 1-based months.
+    // So we must add 1 to `getMonth()` to get the correct calendar month.
+    const startDate = DateTime.fromObject(
+      {
+        year: section.startDate.getFullYear(),
+        month: section.startDate.getMonth() + 1,
+        day: section.startDate.getDate(),
+      },
+      { zone: timeZone }
+    ).startOf('day');
```

Comment:
Good find. Could you encapsulate the conversion of JS Date to Luxon DateTime in a utility function so it can be reused?